src/coreclr/src/debug/daccess/dacimpl.h

```diff
@@ -840,7 +840,8 @@ class ClrDataAccess
       public ISOSDacInterface4,
       public ISOSDacInterface5,
       public ISOSDacInterface6,
-      public ISOSDacInterface7
+      public ISOSDacInterface7,
+      public ISOSDacInterface8
```

Comment:
FYI @davmason is adding this in https://github.com/dotnet/runtime/pull/37853 as well.